### PR TITLE
Make this recipe noarch: python

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by isuruf** [comment]
Since we have dropped python 2 support, this can be noarch: python now

**Comment by isuruf** [comment]
@conda-forge-admin, rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,21 +9,21 @@ source:
...
 build:
-  # Uses 2to3 and cannot be noarch.
-  number: 0
+  number: 1
 
 outputs:
   - name: beautifulsoup4
+    build:
+      noarch: python
     script: build_base.bat  # [win]
```
Comment (ocefpaf) [inline recipe/meta.yaml:18]: Should we remove this too?

Comment (isuruf) [reply recipe/meta.yaml:18]: It's fine to keep there for those who want to build the noarch: python package from windows.

**Pull request merged**